modules/virt-listing-vmis-web.adoc

```diff
@@ -0,0 +1,19 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis-web.adoc
+
+[id="virt-listing-vmis-web_{context}"]
+= Listing standalone virtual machine instances using the web console
+
+Using the web console, you can list and view standalone virtual machine instances (VMIs) in your cluster that are not owned by virtual machines (VMs).
+
+[NOTE]
+====
+VMIs that are owned by VMs or other objects are not displayed in the web console. The web console displays only standalone VMIs. If you want to list all VMIs in your cluster, you must use the CLI.
+====
+
+
+.Procedure
+
+. Click *Workloads > Virtualization* from the side menu. A list of VMs and standalone VMIs displays. You can identify standalone VMIs by the dark colored badges that display next to the virtual machine instance names.
```

Comment:
One more thing - this needs to be a bullet point, not a number, because it's one step. You also need to swap out the `>` for `->`.

Reply:
As per Slack chat exchange, this was updated.